src/components/MoreInfoModal/MoreInfoModal.tsx

```diff
@@ -9,58 +9,51 @@ import Button from '@mui/material/Button';
 import Modal from '@mui/material/Modal';
 import Typography from '@mui/material/Typography';
 import * as React from 'react';
+import { useEffect } from 'react';
 import { Slide } from 'react-slideshow-image';
-import photosHouse from 'src/assets/images/House.png';
-import photosMap from 'src/assets/images/Map.png';
-import photosNavLogo from 'src/assets/images/NavLogo.png';
-import photosNoPhoto from 'src/assets/images/nophoto.png';
+import { useApiGet } from 'src/hooks/useApi';
 
 import FacilitiesList from './FacilitiesList';
 import styles from './MoreInfoModal.module.scss';
 import PropertyInformationList from './PropertyInformationList';
 
-const houseImages = [photosHouse, photosNavLogo, photosMap, photosNoPhoto];
-
 type HouseInfo = {
   url: string;
-  caption: string;
 }[];
 
-// Create array with house pohotos for modal
-const slideImages: HouseInfo = [];
-
-houseImages.forEach((houseImage, index) => {
-  slideImages.push({
-    url: houseImage,
-    caption: `Photo ${index}`,
-  });
-});
-
 const slideProperties = {
   autoplay: false,
+  transitionDuration: 500,
 };
 
-const moreInfo = {
-  address: '420 Baker St, London',
-  price: '$299,999',
-};
-
-const propertyInformation = {
-  propertyType: 'House',
-  area: '1200',
-  yearBuilt: '2000',
-  floor: '1',
-  floorsInBuilding: 'House',
-  roomsNumber: '5',
-  bathroomNumber: '2',
-  heating: 'Radiators',
-};
-
-export default function MoreInfoModal() {
+export default function MoreInfoModal(props) {
   const [open, setOpen] = React.useState(false);
   const handleOpen = () => setOpen(true);
   const handleClose = () => setOpen(false);
 
+  const [houseData, setHouseData] = React.useState();
+  const [housePhotos, setHousePhotos] = React.useState([]);
```

Comment:
Also, wasn't there any issues with TS about no types on useStates that you have here? Cuz u have a boolean one, later u have data, so its propably BasicHouseData or something like that and later u have photos which gonna be strings - I presume(urls). It would be good to even put type 'any' so it doesn't scream for types when we merge it